docs/tutorials/visualize-node-metrics.md

```diff
@@ -198,7 +205,7 @@ the substrate builder community know it exists by listing in on
 
 - The Grafana dashboard configuration for the [Polkadot
   network](https://github.com/w3f/polkadot-dashboard).
-- [Grafana Template](https://grafana.com/grafana/dashboards/11784/) for a Substrate Node Template
+- [Grafana Template](https://grafana.com/grafana/dashboards/13759/) for a Substrate Node Template
```

Comment:
The points at the end of the line should probably be consistent

Reply:
updated.